### DISCUSSION TITLE
Conditional Wrapper for Nacks to build with and without

### DISCUSSION THREAD

**Discussion by brconnell4** [nats-io/nack#206] [Ideas]
Could we introduce a feature flag in rbac-jetstream-controller.yml to toggle the use of the RBAC template provided by NACKS? It would be ideal if this flag defaulted to true, like so:

```
rbac:
  enabled: true
```

The feature flag would act as a wrapper for the entire RBAC file.

The reason for this request is that we manage RBAC for our developers in their namespaced environments. When installing this package as part of our umbrella chart, it currently requires container.role.* permissions.

We prefer to manage these permissions ourselves, consistent with how we handle all other RBAC permissions, while allowing NACKS to focus solely on installing its JetStream controller deployment. We understand that these RBAC permissions are necessary, but we would rather not manage them directly through the NACKS Helm chart.

This isnt a hard request and can make a pull request if allowed to provide a simple solution

**Comment by Jarema** [comment]
Thanks for the propsal.
We will get back to you soon.
